### ISSUE TITLE
Session key backup for minimal downtime migration - outage

### ISSUE THREAD

**Issue by kucharskim** [paritytech/polkadot#4081]
Is there anyway to export (backup) and then restore (import) session keys in case of urgent node fail-over of Polkadot or Kusama validator?

We are developing disaster recovery, rapid failover mechanisms and we would like to have a passive Polkadot / Kusama instance which naturally syncs with the chain and is not a validator, but when a real validator node is affected by an outage, we would like to stop active validator, reconfigure standby node to re-use it's own local database directory (`.../db/`), but re-import session key and start validation again.

Is that scenario possible?

**Comment by bkchr** [comment]
The session keys are also just stored in the file system. So, you are able to backup them. If you don't set an explicit keystore path, check the base path for the keystore.

**Comment by kucharskim** [comment]
Yes, @rphmeier I'm aware of that. Is there a way to generate those session keys by hand?
